builtin/core.wren

```diff
@@ -41,6 +41,15 @@ class List is Sequence {
     }
     return result
   }
+
+  contains(element) {
+    for (item in this) {
+      if (element == item) {
```

Comment:
Style nit: how about making this a one-liner:

```dart
if (element == item) return true
```

I like omitting the `{}` for `if` (in both C and Wren) as long as the entire thing fits on one line and there's no `else` clause.